src/common/scripts/vnc_startup.sh

```diff
@@ -72,7 +72,7 @@ if [[ $VNC_VIEW_ONLY == "true" ]]; then
     #create random pw to prevent access
     echo $(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 20) | vncpasswd -f > $PASSWD_PATH
 fi
-echo "$VNC_PW" | vncpasswd -f >> $PASSWD_PATH
+echo "$VNC_PW" | vncpasswd -f > $PASSWD_PATH
```

Comment:
In case of enabled view only mode, [line 73](https://github.com/ConSol/docker-headless-vnc-container/pull/102/commits/f6df4ca402fc1a0ace7f3206ad5eb82c455909bf#diff-86b6838545b746bab857158e2d204a4eR73) already writes a random password for admin access to `~/.vnc/passwd`.

Changing `>>` to `>` would overwrite the generated password, disabling `view only` mode.

Reply:
As I  build my docker image based on this  with docker commit(i know this method is not well but my image is too complex) , i found that ,the env not work,(before the sh run,the password file is already exsit)   then the sh write password file   again  ,  i think we need to del the password file before you first write it. i will pull again tomorrow use this new idea

Reply:
As I build my docker image based on this with docker commit(i know this method is not well but my image is too complex) , i found that ,the env not work,(before the sh run,the password file is already exsit as the sh hav already run once ) then the sh write password file again , i think i need to del the password file in my image ,that will solve my problem without changing this project  ..  it is not a bug as i consided before. thinks.


i think other beginners of docker as i will fall to this problem too ,  can we check the passwd file before we output,? if file exsit ,delete it first.